tests/test_core.py

```diff
@@ -48,5 +50,7 @@ def test_serialize_json_to_file_errors_windows(self):
             excType = PermissionError  # noqa: F821
 
         with self.assertRaises(excType) as exc:
-            otio.core.serialize_json_to_file({}, self.tmpDir)
-        self.assertIsInstance(exc.exception, excType)
+            otio.core.serialize_json_to_file({}, dir())
+        assert (exc.exception, excType)
```

Comment:
I think this should be:
`assert isinstance(exc.exception, excType)`

This line is doing an `assert` on the `tuple`: `(exc.exception, excType)` - which will always pass because a non-empty tuple is "[truthy](https://www.freecodecamp.org/news/truthy-and-falsy-values-in-python/)"

What this assertion is meant to do is test that the exception raised is of the correct type (based on whether it's python 2 or 3) - The [`isinstance` function](https://docs.python.org/3/library/functions.html#isinstance) will check if a given instance is of the provided type (or one of it's sub-types).

You can try this out in an interactive python interpreter:
```python
>>> # Here is an exception instance, and a desired exception type
>>> exception = Exception("This is not an OSError")
>>> excType = OSError
>>> # Let's try asserting the tuple of those two
>>> assert (exception, excType)
<stdin>:1: SyntaxWarning: assertion is always true, perhaps remove parentheses?
>>> # Note that the python interpreter gives us a warning that our assertion will pass no matter what, oh no!
>>> # Let's try it now using the isinstance function
>>> assert isinstance(exception, excType)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError
>>> # Now we get an expected AssertionError because Exception is not of type OSError
>>> # Let's try it again where exception is of the type OSError
>>> exception = OSError("This is an OSError instance!")
>>> assert isinstance(exception, excType)
>>>
>>> # Note that there is no AssertionError raised in the terminal because the assertion passed as expected
```